index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel="stylesheet">
+</head>
+<body>
+    <main>
+        <header>
+            <a href="#">My logo</a>
+            <div>
+             <i class="fas fa-bars"></i>
+            </div>
```

Comment:
- [ ] Please keep in mind that the hamburger button is referring to a list (menu) so it's important to apply the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) concepts and make it have a `<nav>` tag.
Kindly work on that.